aten/src/ATen/AccumulateType.h

```diff
@@ -68,8 +68,10 @@ template <> struct AccumulateType<int64_t, true> { using type = int64_t; };
 template <> struct AccumulateType<bool, true> {using type = bool; };
 template <> struct AccumulateType<Half, false> { using type = float; };
 template <> struct AccumulateType<BFloat16, false> { using type = float; };
+template <> struct AccumulateType<c10::complex<Half>, false> { using type = c10::complex<float>; };
 template <> struct AccumulateType<c10::complex<float>, false> { using type = c10::complex<double>; };
 template <> struct AccumulateType<c10::complex<double>, false> { using type = c10::complex<double>; };
+template <> struct AccumulateType<c10::complex<Half>, true> { using type = c10::complex<float>; };
```

Comment:
required by
https://github.com/pytorch/pytorch/blob/a0cc38eaca46e25f561adf9d5b5655f625c39377/aten/src/ATen/native/cuda/Indexing.cu#L54

which is required by `index_put_with_sort_kernel`

Reply:
`accscalar` in `index_put_with_sort_kernel` should be opmath, in fact, can you please change it there, and remove these additions?

Reply:
Have updated the `index_put_with_sort_kernel` and removed the new entries.